include/swift/AST/Attr.def

```diff
@@ -299,6 +299,10 @@ SIMPLE_DECL_ATTR(_implicitly_unwrapped_optional,
                  ImplicitlyUnwrappedOptional,
                  OnFunc | OnVar | OnSubscript | OnConstructor | RejectByParser, 72)
 
+DECL_ATTR(_optimize, Optimize,
+          OnFunc | OnConstructor | OnDestructor | OnSubscript |
```

Comment:
Theoretically since you can apply this to subscripts, why not computed properties too?

Reply:
I'll add it. This is OnVar, right?